feature-libs/asm/components/asm-session-timer/asm-session-timer.component.ts

```diff
@@ -30,11 +31,11 @@ export class AsmSessionTimerComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit(): void {
-    this.timeLeft = this.getTimerStartDelayInSeconds();
+    this.initTimer();
```

Comment:
When agent reopen his/her laptop, will the page be refreshed? Will the ngOnInit() be called to init timer again?

Reply:
yes, if it refresh the page, the timer will be reset, similar behavior also could be observed in acc

Reply:
In old code,     this.timeLeft = this.getTimerStartDelayInSeconds(); also refresh it back to 10 minutes

Reply:
If refresh page will reset timer, then it should be ok.